libs/genai/langchain_google_genai/chat_models.py

```diff
@@ -1171,11 +1189,20 @@ def _prepare_request(
                 "Must specify at most one of tool_choice and tool_config, received "
                 f"both:\n\n{tool_choice=}\n\n{tool_config=}"
             )
+              
         formatted_tools = None
         if tools:
```

Comment:
we should raise a warning (or even an error?) if we have both `tools` and `code_execution=True`